lib/java/AdapterLibrary/src/test/java/com/vmware/aria/operations/AdapterInstanceTest.java

```diff
@@ -116,6 +101,30 @@ private JsonPrimitive nil() {
             "collection_window", WINDOW
     ));
 
+    private JsonArray arr(List<? extends JsonElement> a) {
+        return new JsonArray(a);
+    }
+
+    private JsonObject obj(Map<String, ? extends JsonElement> o) {
+        return new JsonObject(o);
+    }
+
+    private JsonPrimitive str(String s) {
+        return JsonElementKt.JsonPrimitive(s);
+    }
+
+    private JsonPrimitive bool(boolean b) {
+        return JsonElementKt.JsonPrimitive(b);
+    }
+
+    private JsonPrimitive n(Number i) {
+        return JsonElementKt.JsonPrimitive(i);
+    }
+
+    private JsonPrimitive nil() {
+        return JsonElementKt.JsonPrimitive((Boolean) null);
+    }
+
```

Comment:
I'm curious why you moved these

Reply:
these were moved by my default formatter

Reply:
Ah, that makes sense. Is that just the default IntelliJ settings?

Reply:
Yes, if there are any specific ones for this project I can import them, reformat and resubmit